src/celengine/render.cpp

```diff
@@ -3166,7 +3170,7 @@ void Renderer::buildRenderLists(const Vector3d& astrocentricObserverPos,
             // Compute the irradiance
             // Instead of summing the reflected light from all nearby stars,
             // we just consider the one with the highest irradiance.
-            float irradiance = 1.175494e-38f; // minimum positive float value
+            float irradiance = std::numeric_limits<float>::min();
```

Comment:
(I'm not sure this code would be uses after all, but) I mean here minimum *positive* value, now it would return 1.17549e-38